docs/pages/eas-update/continuous-deployment.mdx

```diff
@@ -44,6 +56,17 @@ To identify differences in fingerprints on different machines or environments:
 
 To exclude files causing the differences, add them to the **.fingerprintignore** file as described in the [`@expo/fingerprint` README](https://www.npmjs.com/package/@expo/fingerprint).
 
-### Debugging with GitHub Actions
+</Collapsible>
+
+<Collapsible summary="How do I debug the commands that are run in the workflow?">
+
+Running your GitHub workflows in debug mode will automatically add the `--debug` flag to the commands run as part of the action, and the output will be available in the workflow run logs.
+
+</Collapsible>
+
+## GitHub Action: expo-github-action
```

Comment:
I think we probably want to stop suggesting this action at some point, I wonder if that point is now to reduce thrash. @Kudo and I were planning to remove the action once the new action is stable.

Reply:
maybe just recommend the [`continuous-deploy-fingerprint`](https://github.com/expo/expo-github-action/tree/main/continuous-deploy-fingerprint) action directly here in the doc.

we can deprecate [`prebuild-build`](https://github.com/expo/expo-github-action/tree/main/preview-build) in favor of the `continuous-deploy-fingerprint`.
however i think we can still keep [`fingerprint`](https://github.com/expo/expo-github-action/tree/main/fingerprint) alive for people without using EAS to have some interesting idea. we also use the fingerprint github action to label PRs at expo/expo repo.

Reply:
open to either, i'll leave this as-is for now since i don't have enough context and i'm mostly just moving stuff around here. added it to the discussion topics for our update sync next week @wschurman